Python/optimizer_bytecodes.c

```diff
@@ -138,6 +142,24 @@ dummy_func(void) {
         }
     }
 
+    op(_CHECK_ATTR_CLASS, (type_version/2, owner -- owner)) {
+        assert(type_version);
+        if (sym_is_a_class(owner) &&
+            sym_matches_type_version(owner, type_version)) {
```

Comment:
This is incorrect. `sym_matches_type_version` matches the type version of the symbol's class, not that of the symbol.

Reply:
Sorry again, this was an old PR and was back when I more freely resolved things.